.gitignore

```diff
@@ -35,3 +36,6 @@ bench/workflows/*
 
 # Annoying Mac file
 *.DS_Store
+
+# Arsen's local server configuration
+conf/arsen*
```

Comment:
What about just something like `conf/custom`? As it stands, everyone who downloads the code will have a special case for `conf/arsen`. :)